packages/core/src/App/Containers/ResetPasswordModal/reset-password-modal.jsx

```diff
@@ -15,8 +15,6 @@ const ResetPassword = ({ logoutClient, verification_code }) => {
         // Error would be returned on invalid token (and the like) cases.
         // TODO: Proper error handling (currently we have no place to put the message)
         if (error_msg) {
```

Comment:
Any reason why we are removing this? 
Because `console.error(msg)` log points are normally used to send errors to TrackJS.